src/UnitSystem.cc

```diff
@@ -29,57 +29,49 @@
 #include "lsst/afw/geom/transformFactory.h"
 #include "lsst/meas/modelfit/UnitSystem.h"
 
-namespace lsst { namespace meas { namespace modelfit {
+namespace lsst {
+namespace meas {
+namespace modelfit {
 
-UnitSystem::UnitSystem(afw::geom::SpherePoint const & position,
-                       std::shared_ptr<const lsst::afw::image::Calib> calibIn,
-                       double flux) {
-    auto scale = 1.0*lsst::afw::geom::arcseconds;
+UnitSystem::UnitSystem(afw::geom::SpherePoint const& position,
+                       std::shared_ptr<const lsst::afw::image::PhotoCalib> photoCalib_, double flux) {
```

Comment:
The underscore looks a little weird here. Maybe rename the internal variable to `_photoCalib`, per the [style guide](https://developer.lsst.io/cpp/style.html#private-class-variables-and-methods-must-have-underscore-prefix)?

Reply:
Except that it's a `struct`, so the internal variable is `public`.

Reply:
Ah. The constructor must have thrown me off. I assume you won't consider calling both `photoCalib`?